Sources/Formworks/FWForm/FWFormViewController.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  FWFormController.swift
+//  
+//
+//  Created by Artur Carneiro on 29/09/20.
+//
+
+import UIKit
+
+public final class FWFormViewController: UIViewController {
+    // - MARK: Properties
+    private lazy var formCollectionView: UICollectionView = {
+        let collectionView = UICollectionView(frame: .zero, collectionViewLayout: setUpCollectionViewLayout())
+        collectionView.translatesAutoresizingMaskIntoConstraints = false
+        collectionView.register(FWFormViewControllerCell.self,
+                                forCellWithReuseIdentifier: FWFormViewControllerCell.identifier)
+        collectionView.delegate = self
+        collectionView.dataSource = self
+        collectionView.backgroundColor = .fwFormBackground
+        return collectionView
+    }()
+
+    //- MARK: Init
+    /// Initializes a new instance of this type.
+    public init() {
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    //- MARK: Life cycle
+    public override func loadView() {
+        super.loadView()
+        view.backgroundColor = .fwFormBackground
+    }
+    
+    public override func viewDidLoad() {
+        super.viewDidLoad()
+        setUpCollectionViewConstraints()
+    }
```

Comment:
@VicFalcetta why did you split the controller configurations between loadView and viewDidLoad?

Reply:
In this case is just to separate what is needed to set up the View Controller (such as background color) and what is needed to be set up for its subviews (in this case the collection view).

Reply:
Alright. I get it now.
Thanks for the clarification!